tabular/src/autogluon/tabular/models/tabular_nn/tabular_nn_model.py

```diff
@@ -384,7 +384,7 @@ def train_net(self, train_dataset, params, val_dataset=None, initialize=True, se
         self.params_trained['num_epochs'] = best_val_epoch + 1
         return
 
-    def _predict_proba(self, X, preprocess=True):
+    def _predict_proba(self, X, **kwargs):
         """ To align predict wiht abstract_model API.
             Preprocess here only refers to feature processing stesp done by all AbstractModel objects,
```

Comment:
fix typo

Reply:
Got it!

Reply:
Hopefully got them for good, no stesp/wiht now. I'll leave this open until I see the doc show up.